modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java

```diff
@@ -325,14 +326,22 @@ public void updated(Dictionary<String, ?> properties) throws ConfigurationExcept
     ingestZipJobLoad = LoadUtil.getConfiguredLoadValue(properties, ZIP_JOB_LOAD_KEY, DEFAULT_INGEST_ZIP_JOB_LOAD,
             serviceRegistry);
     // try to get overwrite series option from config, use default if not configured
+    isOverwrite = defaultOverWriteValue;
     try {
-      isOverwriteSeries = Boolean.parseBoolean(((String) properties.get(PROPKEY_OVERWRITE_SERIES)).trim());
+      String overwriteParam = (String) properties.get(PROPKEY_OVERWRITE);
+      if (overwriteParam != null) {
+        isOverwrite = BooleanUtils.toBoolean(overwriteParam.trim().toLowerCase(), "true", "false");
+      } else {
+        logger.info("Did not find '{}' in configuration, using default value '{}'", PROPKEY_OVERWRITE, isOverwrite);
+      }
     } catch (Exception e) {
-      isOverwriteSeries = defaultIsOverWriteSeries;
-      logger.warn("Unable to update configuration. {}", e.getMessage());
+      isOverwrite = defaultOverWriteValue;
+      logger.warn("Unable to use '{}' from Ingest configuration file, expected 'true' or 'false'. {}",
+              PROPKEY_OVERWRITE,
+              e.getMessage());
```

Comment:
I would probably log an error and definitely return a proper stack trace instead of just the message: Especially since you are catching just all kinds of errors

```java
logger.warn("Unable to use '{}' from Ingest configuration file, expected 'true' or 'false'.", PROPKEY_OVERWRITE, e);
```

Reply:
Thinking about this again: Fail early and don't accept broken configurations: Just don't catch the error. Having a broken configuration may lead to all kinds of weird behavior and having people realize and fix that right away may be the best solution.

Reply:
I agree! done.